tests/vector/test_vector_env.py

```diff
@@ -58,3 +62,113 @@ def test_custom_space_vector_env():
 
     assert isinstance(env.single_action_space, CustomSpace)
     assert isinstance(env.action_space, Tuple)
+
+
+@pytest.mark.parametrize("base_env", ["Pendulum-v1", "CartPole-v1"])
```

Comment:
Why parametrize only over the two envs instead of all of them?

Reply:
You mean over all gym envs?
I mean, sure, I'm all for it, but the current vectorenv tests (i.e. the only test above) is only using CartPole-v1

Reply:
Added test parametrized with all classic_control + toy_text envs in 9c0e308cc03aff71ffca12105f9e9f5085451d97

Reply:
Hey @RedTachyon is this good? (testing with all envs where `should_skipp_env_spec_for_test = False`)?
As-is, the parametrization of this test generates 579 tests, which take about 1:20 to run on my end (with pytest xdist and 4 parallel workers). I think this will probably take something like 5 minutes to run on GitHub, depending on the machine's hardware.

(here's a link for that function btw: https://github.com/lebrice/gym/blob/e913bc81b83b43ae8ca9b3a02c981b74d31017ea/tests/envs/spec_list.py#L20)